### PR TITLE
Feature/remove palaemon template

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

**Review by mcornaton — APPROVED** [review]
Thanks!

File: build.gradle [hunk 1]
```diff
@@ -177,57 +177,3 @@ task buildImage(type: Exec) {
         return standardOutput.toString()
     }
 }
-
-task buildTrustedImage(type: Exec) {
```
Comment (jeremyjams) [inline build.gradle:181]: It might worth it to call docker/sconify.sh here

Comment (jbern0rd) [reply build.gradle:181]: Do you mean keep a `buildTrustedImage` task calling `docker/sconify.sh` ?

Comment (jeremyjams) [reply build.gradle:181]: Yes :+1:

Comment (jbern0rd) [reply build.gradle:181]: done in 246d87d

Comment (jeremyjams) [reply build.gradle:181]: Very nice, thank you for your time :pray:

**Review by jeremyjams — APPROVED** [review]
Thank you for this PR!

**Pull request merged**